administrator/components/com_localise/language/en-GB/com_localise.ini

```diff
@@ -36,6 +36,13 @@ ADMINISTRATOR="Administrator"
 SITE="Site"
 INSTALLATION="Installation"
 
+[Ajax]
+
+COM_LOCALISE_UPDATE_TRANSLATIONS_LIST_TASK_ERROR="Unable to update the Translations list."
+COM_LOCALISE_UPDATE_TRANSLATIONS_LIST_TASK_FLASH_ERROR="Error updating Translations list"
+COM_LOCALISE_UPDATE_TRANSLATIONS_LIST_TASK_FLASH_SUCCESS="Translations list has been updated"
+COM_LOCALISE_UPDATE_TRANSLATIONS_LIST_TASK_SUCCESS="The file list for the selected language has been updated. Before 'Save' it would be advisable to check that the name of the package and the name of the selected language correspond."
```

Comment:
suggest

`"The file list for the selected language has been updated. Make sure that the package name and the selected language correspond before saving."`

Reply:
Done